fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -224,8 +261,15 @@ public void parseArgs(String[] args) {
                     throw new IllegalArgumentException("missing value for --db-type");
                 }
                 break;
+            case "--bootstrap-schema":
```

Comment:
needs documentation.  probably in the README

Reply:
done